src/GNDStk/Node/test/call.test.cpp

```diff
@@ -54,12 +54,284 @@ struct isotope_t {
 
 inline void convert(const Node<> &n, isotope_t &iso)
 {
-   iso.symbol = n(mixed::meta::symbol);
-   iso.A = n(mixed::meta::A);
-   iso.nuclides = &n(mixed::child::nuclides);
+   iso.symbol = n(misc::meta::symbol);
+   iso.A = n(misc::meta::A);
+   iso.nuclides = &n(misc::child::nuclides);
 }
 
 
+// -----------------------------------------------------------------------------
+// values_string
+// -----------------------------------------------------------------------------
+
+static const std::string values_string =
+R"***(1e-05
+460000
```

Comment:
Can you reduce the number of numbers here? I don't think we need all of these to adequately test these capabilities.

Reply:
Yes, no problem. I'd already reduced it from the full list of values in the file, but can reduce it further. Perhaps by about a factor of two?